container/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node
+EXPOSE 3000
+
+RUN apt-get update && \
+    apt-get -y install jq && \
+    apt-get clean
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+RUN curl -L  https://github.com/IBM/watson-conversation-slots-intro/archive/master.tar.gz --output /tmp/master.tar.gz && \
```

Comment:
usually the expectation is that you've cloned this repo down already so you'd use `ADD . /usrc/src/app` rather than the `run curl`, especially since you're later adding `.sh` from the local repo.

Reply:
We have this in a subdirectory, and cannot 'add .. /usr/src/app' due to permission issues.
So, we could move all the container stuff to the top level and change this....

Reply:
++

Reply:
Everything is moved to top level directory. Done.